packages/openapi3/test/openapi-output.test.ts

```diff
@@ -249,6 +249,23 @@ describe("openapi3: extension decorator", () => {
     strictEqual(oapi.components.parameters.PetId["x-parameter-extension"], "foobaz");
   });
 
+  it("adds an extension to a namespace", async () => {
+    const oapi = await openApiFor(`
+      @extension("x-model-extension", "foobar")
+      @route("/createSong")
+      namespace Suno {
```

Comment:
Is this test actually passing I would have expected it to be annotated with a @service